src/pages/covid_resources.mdx

```diff
@@ -0,0 +1,114 @@
+import { Button } from "../components/button";
+
+![image](Images/rainbow.jpeg)
+
+# **COVID-19 Resources**
+
+### **Dear oSTEM Family,**
+ 
+The state of the world has changed very quickly in the last few weeks with the COVID-19 pandemic that is currently affecting us as global citizens. As the oSTEM community includes displaced members and affiliates from academic student chapters to international volunteers, it is important to us that we use this space to address your specific needs.
+ 
+Academic and professional chapters are the foundation of oSTEM. Though community has often been defined by the value of in-person connections, we are committed to taking a stand against the public health risk caused by this pandemic.
+ 
+In support of your health and safety, until further notice, **we will cease all in-person oSTEM events and meetings and transition to a fully online community.**
+
+<Button onClick={"https://ostem.org/page/covid-19-response"}>
+  Read our Full Message
+</Button>
+
+### **oSTEM Squad Slack Workspace**
```

Comment:
We actually don't use squad slack, we use discord. We can maybe just change that later.

Reply:
Will leave this section for now for future change to Discord info